### PR TITLE
Javascript2 week3/moein

### PR DESCRIPTION
**Pull request by MoeinHeydari90**
This pull request introduces several new features and improvements to the quiz app. It includes:

- A new page for the quiz app
- A new button for downloading the questions
- A script to store the theme when navigating between pages

### CODE REVIEW

File: scripts/quiz-game.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+function firstPlayerPlus() {
+    const pointsInput = document.getElementById("firstPlayer-points");
+    pointsInput.value = parseInt(pointsInput.value) + 1;
+    checkWinner("firstPlayer-name", pointsInput.value);
+}
+
+function secondPlayerPlus() {
+    const pointsInput = document.getElementById("secondPlayer-points");
+    pointsInput.value = parseInt(pointsInput.value) + 1;
```
Comment (islanderupnorth) [inline scripts/quiz-game.js]: This works but another way to convert a `string` to a `number` is by simply writing `+pointsInput.value + 1` - the plus in front converts it!

File: scripts/theme.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    const themeToggleButton = document.getElementById("theme-toggle");
+    themeToggleButton.addEventListener("click", () => {
+        if (document.body.classList.contains("dark-mode")) {
+            document.body.classList.remove("dark-mode");
+            localStorage.setItem("theme", "");
+        } else {
+            document.body.classList.add("dark-mode");
+            localStorage.setItem("theme", "dark-mode");
+        }
```
Comment (islanderupnorth) [inline scripts/theme.js:16]: Nice touch and really great work especially storing the preference locally!

File: styles/styles.css [hunk 3]
```diff
@@ -273,3 +285,16 @@ footer {
...
+
+@media only screen and (max-width: 600px) {
+    main {
+        flex-direction: column;
+    }
+    section,
+    aside {
+        width: 100%;
+    }
+}
```
Comment (islanderupnorth) [inline styles/styles.css:302]: Great, you've implemented Media Queries

**Review by islanderupnorth — COMMENTED** [review]
Hi Moein,

I'm so glad to see the progress you have made over the coming weeks and the logic you have implemented. I don't have many JS fixes as there are many ways to approach things and your logic works as intended!

From an experience perspective I do think that you should have the player names & score next to the quiz so you can take the quiz and score points on the side.

Really nice with the bonus touches of dark mode, media queries and the final sound 🎉

File: scripts/quiz-game.js (hunk 1 above)
Comment (MoeinHeydari90) [reply scripts/quiz-game.js]: such a good idea 👍 I'll use it later

**Pull request merged**